iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/helplets.inl

```diff
@@ -170,17 +170,25 @@ inline bool doesEndWithPathSeparator(const string<StringCapacity>& name) noexcep
     return false;
 }
 
-template <typename F, typename T>
-inline constexpr T from(const F) noexcept
+template <typename SourceType, typename DestinationType>
+inline constexpr DestinationType from(const SourceType value)
```

Comment:
Why does `into` has a `noexcept` qualifier but `from` not?

Reply:
@dkroenke Good catch. As already decided in #1032, we will in the long run only add `noexcept` to the methods where we can guarantee that no exception will be thrown. Here, in a generic template we can guarantee it, so I removed `noexcept`.